### PR TITLE
Add 'Get help' and enhance introduction.

### LINKED ISSUE #4786: Write support information pages for Extensions and Web Store Docs
We currently have nothing on our websites that tell users where to go for help.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #4786

Changes proposed in this pull request:

- Adds a "Get help" page and a new item to the TOC.
- Moves get help from landing page to give it more prominence.
- Replaces moved text with links to "Get help" and "What's new".
- Enhances "What's new" page to reference our other channels.

### CODE REVIEW

**Comment by matthiasrohmer** [comment]
Replicating the answer here to have it as a reference: the reason why your change (the new 'Get help' page) doesn't show in the preview environments is that they are using the production configuration to build. As the Get help page has a date set in the future it is excluded from a production build as it is seen as a scheduled post. Also see: https://developer.chrome.com/docs/handbook/how-to/schedule-a-post/

**Review by AmySteam — APPROVED** [review]
@jpmedley Looks great! 💯

**Comment by jpmedley** [comment]
That seems obvious now.
Joe Medley | Technical Writer, Chrome DevRel | ***@***.*** |
 816-678-7195
*If an API's not documented it doesn't exist.*

File: site/en/docs/extensions/gethelp/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: 'Where to go for help with questions not covered by the documentation.'
+date: 2023-02-21
+tags:
+  - extensions
+---
+
+We do our best to improve and expand the content on this site. Some of you might run into problems we don't cover. Fortunately, there are other ways for you to get help.
+
+- Many members of the Chrome extensions team monitor and participate in the [Chromium extensions](https://groups.google.com/a/chromium.org/g/chromium-extensions) Google Group.
+- StackOverflow has a few tags you should follow and search. The [google-chrome-extension](https://stackoverflow.com/questions/tagged/google-chrome-extension) tag covers Chrome extensions generally. There's also a [chrome-extension-manifest-v](https://stackoverflow.com/questions/tagged/chrome-extension-manifest-v3) tag.
```
Comment (AmySteam) [inline site/en/docs/extensions/gethelp/index.md:13]:
```suggestion
- StackOverflow has a few tags you should follow and search. The [google-chrome-extension](https://stackoverflow.com/questions/tagged/google-chrome-extension) tag covers Chrome extensions generally. There's also a [chrome-extension-manifest-v3](https://stackoverflow.com/questions/tagged/chrome-extension-manifest-v3) tag.
```

**Pull request merged**